project-base/storefront/components/Pages/Order/TransportAndPayment/helpers.ts

```diff
@@ -317,12 +317,12 @@ export const useLoadTransportAndPaymentFromLastOrder = (
     };
 
     useEffect(() => {
-        if (!isUserLoggedIn) {
+        if (!isUserLoggedIn || !cart) {
```

Comment:
Hmm, if I understand this issue properly what about solve it by using `useIsUserLoggedIn` hook which we would modify to return 3 states - true, false and undefined. And then distinguish between them to know if user API is already returned or if user is really not logged in?

Reply:
Well, what would that solve? I need to wait for the cart to be loaded. Not sure how your suggestion would fix that.